### PR TITLE
update github action file

### PR DESCRIPTION
**Pull request by senthil10**

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -1,38 +1,78 @@
...
+        uses: docker/build-push-action@v3
+        with:
+          file: ${{ matrix.dockerfile }}
           context: .
           push: true
-          tags: |
-            scilifelabdatacentre/covid-portal:${{ steps.docker-tag.outputs.tag }}
-            ghcr.io/scilifelabdatacentre/covid-portal:${{ steps.docker-tag.outputs.tag }}
+          # Set to wanted target, or remove if you do not use targets
+          target: production
```
Comment (talavis) [inline .github/workflows/dockerhub.yml:76]:
```suggestion
```

**Review by talavis — CHANGES_REQUESTED** [review]
I didn't see anything that could be an issue except the `target` line (I don't think the covid portal Dockerfile has targets).

**Review by talavis — APPROVED** [review]

**Comment by senthil10** [comment]
@talavis have removed it now, tack 👍

**Comment by senthil10** [comment]
Tack @talavis :)

**Pull request merged**